apps/web/pages/booking/[uid].tsx

```diff
@@ -563,7 +563,8 @@ export default function Success(props: SuccessProps) {
                         </div>
                       </>
                     )}
-
+                  </div>
+                  <div className="text-bookingdark dark:border-darkgray-200 mt-8 text-left dark:text-gray-300">
```

Comment:
I dont think booking dark as a light mode value 🤔 as its the contrast value caluclated for the darkmode as a background 

In theory i can set darkmode to a colour that constrast value is white and this text would be very hard to read. 

<img width="813" alt="CleanShot 2023-03-21 at 23 11 37@2x" src="https://user-images.githubusercontent.com/55134778/226650854-d972f5ca-7f5d-4adf-9982-46f6a9f400df.png">

```suggestion
                  <div className="text-gray-300 dark:border-darkgray-200 mt-8 text-left dark:text-darkgray-300">
```

Reply:
@sean-brydon text-bookingdark is used everywhere in /booking/[uid] page